src/fmdb/FMDatabase.m

```diff
@@ -826,7 +843,9 @@ - (FMResultSet *)executeQuery:(NSString *)sql withArgumentsInArray:(NSArray*)arr
                 idx++;
             }
             else {
-                NSLog(@"Could not find index for %@", dictionaryKey);
+                if (_logsErrors) {
```

Comment:
Without this NSLog, the user won't know she's sending an unknown parameter name.